flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/io/CheckpointedInputGate.java

```diff
@@ -233,15 +234,17 @@ public long getLatestCheckpointId() {
 	 *
 	 * @return The duration in nanoseconds
 	 */
-	public long getAlignmentDurationNanos() {
+	@VisibleForTesting
```

Comment:
? It's not used anywhere in the tests, isn't it?

Reply:
It is actually used in many tests in `CheckpointBarrierAlignerTestBase`. I ever tried to refactor these tests to use these two methods from `CheckpointBarrierHandler` directly, then we can remove them from `CheckpointedInputGate`. But it would bring more refactoring work after checking the codes for exposing the `CheckpointBarrierHandler` while constructing the gate. So I still keep the current way now.

Reply:
Ops, sorry for the confusion. I thought that you were changed this from private to `@VisibleForTesting` for some test that you didn't commit, while it was `public` before. Please ignore my comment.